example/package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "perfecto-cypress-example",
   "version": "0.0.1",
+  "scripts": {
```

Comment:
this is a code example for costumers, so it should remind clean and we don't want to expose this internal option to change NASE_BACKEND_URL
i think it is better. to not push changes in this file